### PR TITLE
IGNITE-14682 Fix sending messages from a node to itself

### PR DESCRIPTION
**Pull request by sashapolo**
https://issues.apache.org/jira/browse/IGNITE-14682

### CODE REVIEW

File: modules/network/src/integrationTest/java/org/apache/ignite/network/scalecube/ITScaleCubeNetworkMessagingTest.java [hunk 1]
```diff
@@ -48,6 +48,11 @@
 
 /** */
 class ITScaleCubeNetworkMessagingTest {
+    /**
+     * Test cluster.
+     * <p>
```
Comment (SammyVimes) [inline modules/network/src/integrationTest/java/org/apache/ignite/network/scalecube/ITScaleCubeNetworkMessagingTest.java:53]: `p` should have a closing tag

**Review by SammyVimes — APPROVED** [review]
I'm generally ok with suggested changes, just wanted to point out a minor javadoc issue

File: modules/network/src/integrationTest/java/org/apache/ignite/network/scalecube/ITScaleCubeNetworkMessagingTest.java (hunk 1 above)
Comment (sashapolo) [reply modules/network/src/integrationTest/java/org/apache/ignite/network/scalecube/ITScaleCubeNetworkMessagingTest.java:53]: why?

Comment (SammyVimes) [reply modules/network/src/integrationTest/java/org/apache/ignite/network/scalecube/ITScaleCubeNetworkMessagingTest.java:53]: Hm, I always thought that p requires a closing tag or at least it should be self-closing. Ok then)

**Pull request merged**